packages/tools/viewer-alpha/src/viewer.ts

```diff
@@ -292,13 +296,13 @@ export class Viewer implements IDisposable {
 
         // TODO: Disable audio for now, later figure out how to re-introduce it through dynamic imports.
         options = {
-            ...options,
+            ...(options ?? {}),

```

Comment:
the options management gets pretty hard to read.

Reply:
Yea merging options is pretty verbose. Since they are plain js objects, I'm not sure how to do this better, other than using an object merging library. That said, this specific code should go away once we support dynamically importing the audio engine. Also regardless of that, I'm pretty sure I will change this class more later to not even pass through `LoadAssetContainerOptions`. I think it will be better for this class to be more stateful and mimic the lifetime of the DOM layer to simplify the "binding" of the DOM layer (and any future other UI layers) to this Viewer layer.